test/data/settlements/{settlementId}/participants/{participantId}.js

```diff
@@ -0,0 +1,115 @@
+/*****
```

Comment:
Why are you are re adding the mocked tests that you removed on the last PR?

Reply:
It is required in src/handlers/settlements/{settlementId}/participants/{participantId}.js